arex-schedule-web-api/src/main/java/com/arextest/schedule/service/ReplayCaseTransmitService.java

```diff
@@ -229,6 +232,11 @@ void updateSendResult(ReplayActionCaseItem caseItem, CaseSendStatusType sendStat
             caseItem.setTargetResultId(StringUtils.EMPTY);
         }
         caseItem.setSendStatus(sendStatusType.getValue());
+        if (sendStatusType == CaseSendStatusType.SUCCESS) {
+            AsyncCompareCaseTaskRunnable compareTask = new AsyncCompareCaseTaskRunnable(caseItem);
+            compareExecutorService.execute(compareTask);
+            LOGGER.info("Async compare task distributed, case id: {}", caseItem.getId());
+        }
         if (sendStatusType == CaseSendStatusType.SUCCESS && replayResultComparer.compare(caseItem, true)) {
```

Comment:
The original logic has not been removed, and it seems to have been compared twice